src/main/java/nextstep/qna/domain/Answer.java

```diff
@@ -47,11 +48,15 @@ public Long getId() {
         return id;
     }
 
-    public Answer setDeleted(boolean deleted) {
+    private Answer setDeleted(boolean deleted) {
```

Comment:
```suggestion
    private Answer delete() {
        this.deleted = true;
    }
```
위와 같이 구현하거나 setter 메서드를 아예 제거하는 것은 어떨까?